src/coreclr/src/jit/CMakeLists.txt

```diff
@@ -10,6 +10,124 @@ endif()
 
 add_compile_options($<$<CXX_COMPILER_ID:MSVC>:-W4>)
 
+function(remove_standard_jit_targetdetails)
```

Comment:
We should be able to drive this via target properties and generator expressions instead of having to use the nested folder scopes.